kafka-ui-react-app/src/components/Topics/Topic/ConsumerGroups/TopicConsumerGroups.tsx

```diff
@@ -48,7 +48,7 @@ const TopicConsumerGroups: React.FC = () => {
         enableSorting: false,
       },
       {
-        header: 'Messages Behind',
+        header: 'consumer lag',
```

Comment:
Please start it with an upper case letter as well

Reply:
Done